commit e7cf058cbba3
Author: Alfredo.Yang <ayang@mozilla.com>
Date:   Tue Aug 29 15:33:26 2017 +0800

    Bug 1377278 - accept 'matroska' as webm doctype. r=kinetik
    
    MozReview-Commit-ID: 3SCOrecahPB
    
    --HG--
    extra : rebase_source : c94cbc48f0d850f9df19fe4c85066ca594164853
---
 media/libnestegg/README_MOZILLA | 2 +-
 media/libnestegg/src/nestegg.c  | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git media/libnestegg/README_MOZILLA media/libnestegg/README_MOZILLA
index 384a2874d137..979b4e936dfe 100644
--- media/libnestegg/README_MOZILLA
+++ media/libnestegg/README_MOZILLA
@@ -5,4 +5,4 @@ Makefile.in build files for the Mozilla build system.
 
 The nestegg git repository is: https://github.com/kinetiknz/nestegg
 
-The git commit ID used was 369d153d5d176f58e2742fc843b3ed134d7c64f4.
+The git commit ID used was 017f480db78bcdc607e6dba36cef806b793edf2c.
diff --git media/libnestegg/src/nestegg.c media/libnestegg/src/nestegg.c
index b9283c31f5b2..133732e481bc 100644
--- media/libnestegg/src/nestegg.c
+++ media/libnestegg/src/nestegg.c
@@ -2115,7 +2115,7 @@ nestegg_init(nestegg ** context, nestegg_io io, nestegg_log callback, int64_t ma
 
   if (ne_get_string(ctx->ebml.doctype, &doctype) != 0)
     doctype = "matroska";
-  if (strcmp(doctype, "webm") != 0) {
+  if (!!strcmp(doctype, "webm") && !!strcmp(doctype, "matroska")) {
     nestegg_destroy(ctx);
     return -1;
   }
